digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/AssetDataImporter.scala

```diff
@@ -858,4 +858,201 @@ class AssetDataImporter {
     }
     props
   }
+
+  /**
+    * Get address information to mass transit stop assets from VVH road link (DROTH-221).
+    *
+    * @param vvhRestApiEndPoint
+    */
+  def getMassTransitStopAddressesFromVVH(vvhRestApiEndPoint: String) = {
+    val vvhClient = new VVHClient(vvhRestApiEndPoint)
+    withDynTransaction {
+      println("Phase 1 out of 2. Getting stops with both names missing")
+      val municipalities = getMTStopsMunicipalitycodeBothMissing()
+      municipalities.foreach { municipalityCode =>
+        val startTime = DateTime.now()
+        println(s"*** Processing municipality: $municipalityCode")
+        val listOfStops = getMTStopsWOAddresses(municipalityCode)
+        val roadLinks = vvhClient.fetchByMunicipality(municipalityCode)
```

Comment:
It's not better to get the roadlinks by the link ids we have on "listOfStops" instead of getting all roadlinks for one municipality?